api/s2n.h

```diff
@@ -1612,10 +1616,15 @@ S2N_API
 extern int s2n_connection_prefer_low_latency(struct s2n_connection *conn);
 
 /**
- * Provides a smooth transition from s2n_connection_prefer_low_latency() to s2n_connection_prefer_throughput().
+ * Changes the behavior of s2n-tls when sending data to initially prefer records
+ * small enough to fit in single ethernet frames.
+ *
+ * The connection will send the first resize_threshold bytes in records small enough to
+ * fit in a single standard 1500 byte ethernet frame. Later, whenever timeout_threshold seconds
+ * pass without sending data, the connection will revert to this behavior and send small records again.
```

Comment:
Would it make sense to add a sentence here describing what happens to the record size after `resize_threshold` bytes is sent and before `timeout_threshold` seconds is reached? Or can this be assumed/read in the usage guide?

Reply:
Writing that without referencing every other fragment length related API is tricky :\ I tried a rewrite, see if that's any better.